src/components/utils/CalciteExpandToggle.tsx

```diff
@@ -28,12 +28,15 @@ function getClosestShellLayout(el: HTMLElement): CalciteLayout | null {
 
 export function toggleChildActionText({
   parent,
+  expand,
   expanded
 }: {
   parent: HTMLElement;
+  expand: boolean;
   expanded: boolean;
 }): void {
-  parent.querySelectorAll("calcite-action").forEach((action) => (action.textEnabled = expanded));
+  expand &&
```

Comment:
Instead of modifying this utility, could you decide whether to use `toggleChildActionText` at its call site based on `expand` (or some other condition)?